requirements.txt

```diff
@@ -0,0 +1,10 @@
+## -*- coding: utf-8 -*-
+# 
+# Copyright (c) 2016, Veritomyx, Inc.
+#
+# This file is part of the Python SDK for PeakInvestigator
+# (http://veritomyx.com) and is distributed under the terms
+# of the BSD 3-Clause license.
+requests
+paramiko
+simplejson
```

Comment:
This also appears to already exist: https://github.com/Veritomyx/PeakInvestigator-Python-SDK/blob/master/requirements.txt